### DISCUSSION TITLE
Cannot install even though /bin/sh links to bash

### DISCUSSION THREAD

**Discussion by wiraki** [starship/starship#3947] [Q&A]
I get: 

x Running installation script with non-POSIX `bash` may cause errors.
x Please use `sh` instead.

Not sure how to work around this?

**Comment by chipbuster** [comment]
The POSIX check looks at two things:

- Whether `BASH_VERSION` exists
- Whether `POSIXLY_CORRECT` exists

I was under the impression that `bash` would automatically launch in POSIX mode if it found that `argv[0]` was `sh`, but since that's not the case, you should be able to emulate this by running it with the `--posix` flag.
